.github/workflows/docker.yml

```diff
@@ -34,3 +67,27 @@ jobs:
           DOCKER_USER: github
           DOCKER_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
         run: make push-manifest-list
+      - name: Login to Stackable Harbor
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d # tag=v3
+        with:
+          registry: oci.stackable.tech
+          username: robot$sdp+github-action-build
+          password: ${{ secrets.HARBOR_ROBOT_SDP_GITHUB_ACTION_BUILD_SECRET }}
+      - name: Set up Cosign
+        uses: sigstore/cosign-installer@9614fae9e5c5eddabb09f90a270fcb487c9f7149 # tag=v3.3.0
+      - name: Build Harbor Manifest List
+        shell: bash
+        env:
+          DOCKER_USER: github
+          DOCKER_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
+        run: |
+          make build-manifest-list REPO=oci.stackable.tech/sdp
+          MANIFEST_LIST_NAME=oci.stackable.tech/sdp/ubi8-rust-builder
+          # docker manifest push returns directly the sha of the manifest list
+          # As it is an experimental feature, this might change in the future
+          # Further reading: https://docs.docker.com/reference/cli/docker/manifest/push/
+          DIGEST=$( docker manifest push "$MANIFEST_LIST_NAME:latest")
```

Comment:
@Maleware Maybe remove `manifest_push` in the `Makefile` if it's not used anymore? Or use the `make` command here.

Reply:
Done, thanks again :)